server/src/main/java/org/apache/iotdb/db/mpp/plan/analyze/StandalonePartitionFetcher.java

```diff
@@ -129,6 +130,11 @@ public DataPartition getOrCreateDataPartition(
     }
   }
 
+  @Override
+  public boolean updateRegionCache(TRegionRouteReq req) {
+    return false;
```

Comment:
The false result here combine with CACHE_UPDATE_FAIL might bewilder for ConfigNode